gemp-lotr/gemp-lotr-async/src/main/web/ban.html

```diff
@@ -1,2 +1,49 @@
+<link rel="stylesheet" type="text/css" href="css/dark-hive/jquery-ui-1.8.16.custom.css">
```

Comment:
i'm turning this page into a sandbox. No changes here need to be reviewed, but it's a fair pushback to say "let's move this to a sandbox.html"